README.md

```diff
@@ -1,8 +1,7 @@
-<p align="center">
-  <img src="./program_info/polymc-header-black.svg#gh-light-mode-only" alt="PolyMC logo"/>
-  <img src="./program_info/polymc-header.svg#gh-dark-mode-only" alt="PolyMC logo"/>
+ <p align="center">
+<img src="./program_info/polymc-header-black.svg#gh-light-mode-only" alt="PolyMC logo"/ width="50%">
+<img src="./program_info/polymc-header.svg#gh-dark-mode-only" alt="PolyMC logo"/width="50%">
```

Comment:
I don't have any preference about this change, but the original lines use self-closing tags, so please keep the slash position to last.
```suggestion
<img src="./program_info/polymc-header-black.svg#gh-light-mode-only" alt="PolyMC logo" width="50%"/>
<img src="./program_info/polymc-header.svg#gh-dark-mode-only" alt="PolyMC logo" width="50%"/>
```